org.eclipse.buildship.ui.import/plugin.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="org.eclipse.e4.ui.importer.projectConfigurators">
+      <projectConfigurator
+            class="org.eclipse.buildship.ui.smartimport.internal.GradleProjectConfigurator"
+            label="%gradle.importer.label">
+         <activeWhen>
+            <hasFile
+                  path="build.gradle">
```

Comment:
This wouldn't work if the build script is named differently or is written in Kotlin and is named `build.gradle.kts`.
Relying on `settings.gradle` could become an issue too once it gets possible to write settings scripts in Kotlin in a `settings.gradle.kts` file.